pkg/cri/server/sandbox_run_windows.go

```diff
@@ -67,6 +67,7 @@ func (c *criService) sandboxContainerSpec(id string, config *runtime.PodSandboxC
 		customopts.WithAnnotation(annotations.SandboxNamespace, config.GetMetadata().GetNamespace()),
 		customopts.WithAnnotation(annotations.SandboxName, config.GetMetadata().GetName()),
 		customopts.WithAnnotation(annotations.SandboxLogDir, config.GetLogDirectory()),
+		customopts.WithAnnotation(annotations.WindowsHostProcess, config.GetSecurityContext().GetHostProcess())
```

Comment:
Does this compile? I thought this took a string for the second param

Reply:
We'd just have to convert the actual bool into a string representation

Reply:
it seems to build ok, but maybe the tests fail, but yes if it needs a string then you're right ill need to convert